ocpp/v201/enums.py

```diff
@@ -11,6 +13,13 @@ class StrEnum(str, Enum):  # pragma: no cover
 class Action(StrEnum):
     """An Action is a required part of a Call message."""
 
+    def __init__(self, *args, **kwargs):
+        warn(
+            "Action enum contains deprecated members and will be removed in "
+            "the next major release, please use snake case members."
+        )
```

Comment:
Can you apply the comment I made [here](https://github.com/mobilityhouse/ocpp/pull/600#discussion_r1484154853) to this PR?

Reply:
Satisfied with commit [50e4a95](https://github.com/mobilityhouse/ocpp/pull/580/commits/50e4a955cb6418e703fd5e0ce52f23d94a9b714c)